### ISSUE TITLE
Demonstrate how to use the trained model with new data in examples

### ISSUE THREAD

**Issue by zya** [tflearn/tflearn#525]
This is more of a feature request around examples. 

It'd be very useful to extend the examples to demonstrate how one might use the trained models on new data. This is already done for the generative models such as the NLP City Name Generator but when it comes to the classifiers the examples are currently only concerned with creating networks and training. They never show how the model can be used on new data.

For example, the `lstm.py` currently finishes with the line where `mode.fit` is called. What I'm suggesting is to extend the example code to include a case where the model is used on new data. 
````python
.....
# Training
model = tflearn.DNN(net, tensorboard_verbose=0)
model.fit(trainX, trainY, validation_set=(testX, testY), show_metric=True,
          batch_size=32)

# Use
new_sentence = 'this is a new sentence to be analysed using the trained model`
# code to prepare the new string ....
predictions = model.predict(new_sentence)
print(predictions)
```` 

Same goes for the computer vision examples. 
This can be particularly useful for people (like myself) who are new to machine learning.

**Comment by aymericdamien** [comment]
Yes, I agree! We need more examples to show users how to use their own data. We can add this to our roadmap. I mark this thread as 'contribution welcome' in case anyone wanna help :)

**Comment by nebgru** [comment]
I have been using a model I trained on my own data, but I'm getting conflicting results, or should I say I do not understand the statistics I'm getting. Sorry if the remark is not in line with this thread.
When I train the model, I get stats like
Training Step: 1320  | total loss: 0.37131 | time: 9.274s
| Adam | epoch: 020 | loss: 0.37131 - acc: 0.9663 | val_loss: 0.09159 - val_acc: 0.9971 -- iter: 1050/1050
The way I read this is "training accuracy 0.9663 and validation accuracy 0.997
However, when I try applying the model to the test data with the predict function, the best I get is around 70%. What am I missing?

**Comment by keshavnagpal** [comment]
Hi, this is good but I'm getting an error "cannot convert string to float" for the new_sentence variable. I think I need to provide it in form of a vector. how can i convert the string to vector ?

Thank you
